content/docs/1.18/opentelemetry.md

```diff
@@ -110,6 +110,6 @@ spec:
 
 <2> Config field exposes OpenTelemetry collector configuration. This field has been added to collector, agent, ingester and all-in-one CR nodes.
 
-<3> Health check port has to match Jaeger component health check port from the [deployment page](deployment.md).
+<3> Health check port has to match Jaeger component health check port from the [deployment page](../deployment).
```

Comment:
I believe these should be `./deployment`.

Reply:
No, because each page is rendered as a separate folder, like this: `/docs/1.19/deployment/`

You can verify via the netlify/jaegertracing/deploy-preview check below.

Reply:
Yup, you're right.  I was just using a markdown viewer.